### PR TITLE
Update changelog for 1.7.0-insiders

### PR DESCRIPTION
**Pull request by Colengms**

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
+* Add a command to restart IntelliSense for a specific file. [#3727](https://github.com/microsoft/vscode-cpptools/issues/3727)
+* Add support for macOS app bundles [#6726](https://github.com/microsoft/vscode-cpptools/issues/6726)
+	* [MIEngine#1091](https://github.com/microsoft/MIEngine/pull/1091)
+* Add support for Go To / Peek Type Definition. [#7999](https://github.com/microsoft/vscode-cpptools/issues/7999)
+
+### Enhancements
+* In generated build tasks, add a compiler arg to cause color to be displayed in gcc/clang output in terminal. [PR #8165](https://github.com/microsoft/vscode-cpptools/pull/8165)
+
+### Bug Fixes
+* Fix an issue with signature help for overloaded constructors. [#1664](https://github.com/microsoft/vscode-cpptools/issues/1664)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md:14]: Are these the same issue?

**Review by sean-mcmanus — APPROVED** [review]

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,41 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 1.7.0-insiders: Sept 24, 2021
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: Sept->September.

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/CHANGELOG.md (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/CHANGELOG.md:14]: 1664 and 4413?

File: Extension/CHANGELOG.md (hunk 2 above)
Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: And probably better to wait till Monday?

**Review by michelleangela — APPROVED** [review]

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**